packages/terra-functional-testing/lib/docker/docker-compose.yml

```
@@ -0,0 +1,39 @@
+version: '3.7'
+
+services:
+  hub:
+   image: selenium/hub:${IMAGE}
```

I'd be more verbose to ensure we avoid conflicts and unexpected ENVs
```suggestion
   image: selenium/hub:${TERRA_IMAGE_VERSION}
```

---

Also you could build the default in here instead of in the selenium service:
```suggestion
   image: selenium/hub:${IMAGE:3.141.59-20200525}
```

---

SELENIUM_DOCKER_VERSION might be an option as well

---

I added the default in the selenium service JavaScript file so I could print it to the console on startup and guarantee it was the correct version and not a hardcoded string. I'll work on renaming the env variables. I'll probably go with Emily's suggestion. I've gone back and forth on the word order. The repo is `docker-selenium` but I always read/say selenium-docker. The repo name sounds backwards to me. 

```
SELENIUM_DOCKER_VERSION
DOCKER_SELENIUM_VERSION
```

---

I'm paranoid so i'd probably throw a terra on there to ensure uniqueness, but i'm paranoid.

---

Renamed to `TERRA_SELENIUM_DOCKER_VERSION`

https://github.com/cerner/terra-toolkit/pull/412/commits/359d9f7c6d9c2a604f058eff2416ce412b9f4edc